Add MEGA & public availability of boost to availability dashboard

weilu opened the pull request:
Also allow extraction of only executed or approved sheet from CCI excel sheets, instead of skipping the country altogether.

yukinko-iwasaki commented on quality/data_availability_dlt.sql:
```diff
@@ -186,8 +192,46 @@ OR REFRESH LIVE TABLE data_availability USING DELTA AS (
+    END AS boost_public,
     t.boost_earliest_year,
     t.boost_latest_year,
+    CASE WHEN m.country_name IS NULL THEN 'No' ELSE 'Yes' END as avail_on_mega,
```
why not put avail_on_mega and boost_public side by side?

weilu replied:
Done!

yukinko-iwasaki commented on quality/extract_cci_approved_executed.py:
```diff
@@ -13,14 +13,17 @@ def get_cci_metadata(prune=False):
+        if sheet_name not in xls.sheet_names:
+            sheet_name = 'Approved'
+
+        df = pd.read_excel(filename, sheet_name=sheet_name, na_values=['..'])
```
Since we already have the ExcelFile object at the line 16,  just use xls.parse(...) or read_excel(xls)?

weilu replied:
Good spotting! Done.

yukinko-iwasaki reviewed (commented):
Let 2 comments.
I also realized that the 
https://w0lxdrconn01.worldbank.org/content/16b911ef-1cae-4f91-883d-3df8471ddb41/availability
is not rendering the table with the a generic error message. Is this the link I am supposed to be checking?

```
<!doctype html>
<html lang=en>
<title>500 Internal Server Error</title>
<h1>Internal Server Error</h1>
<p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>

```

weilu commented:
@yukinko-iwasaki aside from addressing your comments, I also added a conditional exit statement to stop the execution of the rest of the cells if there is no new data to process.

yukinko-iwasaki reviewed (approved):
Looks good to me!
